include/picongpu/fields/FieldTmp.tpp

```diff
@@ -57,11 +57,11 @@ namespace picongpu
         SimulationFieldHelper<MappingDesc>( cellDescription ),
         m_slotId( slotId )
     {
-        m_commTagScatter =
-            ++pmacc::traits::detail::GetUniqueTypeId< uint8_t >::counter +
-            SPECIES_FIRSTTAG;
-        m_commTagGather = ++pmacc::traits::detail::GetUniqueTypeId< uint8_t >::counter +
-            SPECIES_FIRSTTAG;
+        /** \todo this is a work-around to get a unique tag,
+         *  potentially unsafe in case there are too many type ids
```

Comment:
maybe be more specific what `too many type ids` means? :)
Larger than `SPECIES_FIRSTTAG`?

Reply:
Made me think of that and realize it is actually not a problem. Updated the comment.